dev/com.ibm.ws.security.registry.basic/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -50,11 +50,11 @@ member.name.desc=Name of a user in a Basic User Registry group.
 
 #==== BASIC X.509 CERTIFICATE MAPPING ====
 certificate.map.mode=Certificate map mode
-certificate.map.mode.desc=Specifies the X.509 certificate authentication mapping mode for the basic registry: DISABLED, PRINCIPAL_CN, or CUSTOM.
+certificate.map.mode.desc=Specifies the X.509 certificate authentication mapping mode for the basic registry: PRINCIPAL_CN, CUSTOM, or NOT_SUPPORTED.
 
-certificate.mapper.id=CertificateMapper ID
-certificate.mapper.id.desc=Specifies the CertificateMapper ID to use when the X.509 certificate authentication mapping mode is CUSTOM
+certificate.mapper.id=Certificate mapper ID
+certificate.mapper.id.desc=Specifies the X509CertificateMapper to use when the X.509 certificate authentication mapping mode is CUSTOM. The value must match the value of the 'x509.certificate.mapper.id' property that is specified for the X509CertificateMapper implementation.
 
 certificate.map.mode.principal_cn=The basic registry attempts to authenticate the X.509 certificate by mapping the PrincipalName value in the X.509 certificate to the exact distinguished name (DN) in the repository. If a single matching entity is found, the authentication is successful. If a matching entity is not found or more than a single matching entity is found, authentication fails and the program returns an error.
-certificate.map.mode.custom=The basic registry attempts to authenticate the X.509 certificate by using the custom CertificateMapper implementation specified by certificateMapperId. If a single matching entity is found, the authentication is successful. If a matching entity is not found or more than a single matching entity is found, authentication fails and the program returns an error. 
-
+certificate.map.mode.custom=The basic registry attempts to authenticate the X.509 certificate by using the custom X509CertificateMapper implementation specified by certificateMapperId. If a single matching entity is found, the authentication is successful. If a matching entity is not found or more than a single matching entity is found, authentication fails and the program returns an error. 
```

Comment:
"implementation **that is** specified by..."
Give certificateMapperId a label word after it like you did with **the** X.509 **certificate**.